vendor/github.com/livepeer/go-livepeer-basicnet/basic_network.go

```diff
@@ -118,7 +118,7 @@ func NewBasicVideoNetwork(n *BasicNetworkNode, workDir string, publicIP string,
 		peerCache := NewPeerCache(n.PeerHost.Peerstore(), fmt.Sprintf("%v/conn", workDir))
 		peers := peerCache.LoadPeers()
 		for _, p := range peers {
-			glog.Infof("Connecting to cached peer: %v(%v)", peer.IDHexEncode(p.ID), p.Addrs)
+			glog.V(common.SHORT).Infof("Connecting to cached peer: %v(%v)", peer.IDHexEncode(p.ID), p.Addrs)
```

Comment:
Is this in basicnet?

Reply:
Yeah.  I merged it since it's a small logging change.